plugins/octopus-deploy/CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @backstage/plugin-octopus-deploy
 
+## 0.3.0
+
+### Minor Changes
+
+- Added option to create an extension field for the Octopus Deploy.
+  - Allows the user to select a project group from Octopus Deploy.
+
```

Comment:
Make sure to not direclty adjust the Changelog, but run `yarn changeset` to create a changeset instead, which will than automatically updated the Changelog once approved:)